brambling/templates/brambling/event/organizer/update.html

```diff
@@ -1,9 +1,295 @@
+{% load floppyforms %}
```

Comment:
Alternatively, I could do this too. It sounds fairly easy _and_ useful. I wonder about the best practice for structuring something like that though.